src/pages/components/loading/style.mdx

```diff
@@ -15,33 +31,61 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 |               | background       | `$layer-accent` \* |
 | Page overlay  | background-color | `$overlay`         |
 
-## Structure
+<Caption fullWidth>
+  \* Denotes a contextual color token that will change values based on the layer
+  it is placed on.
+</Caption>
 
-| Element       | Property      | px / rem | Spacing token |
-| ------------- | ------------- | -------- | ------------- |
-| Large spinner | height, width | 88 / 5.5 | –             |
-| Small spinner | height, width | 16 / 1   | –             |
+<Row>
+<Column colLg={8}>
 
-<div className="image--fixed">
+<Tabs>
 
-![Large spinner structure measurements](images/loading-style-1.png)
+<Tab label="Large">
 
-</div>
+![Large loading component color](images/loading-style-color-large.png)
 
-<Caption>
-  Structure measurements for small and large loading spinner | px / rem
-</Caption>
+</Tab>
 
-### Placement
+<Tab label="Small">
 
-The large loading spinner should appear centered over a page or content that it
-is loading. Please note that the top-level navigation is not included in the
-page loading overlay.
+![Small loading component color](images/loading-style-color-small.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+## Typography
+
+Text is not included with the loading indicator by default. If including text,
+it is recommended to use `body-compact-01` with the large indicator or with the
+small indicator. See more specific typography guidance on the inline loading
+[Style](/components/inline-loading/style) tab.
+
+## Size
+
+There are two loading indicator sizes: **large** and **small**. For more
+information about specific use cases for each loading indicator size, see the
+[sizing](/components/loading/usage/#sizing) section on the Usage tab.
+
+| Element   | Size       | Height (px / rem) |
+| --------- | ---------- | ----------------- |
+| Indicator | Large (lg) | 88 / 5.5          |
+|           | Small (sm) | 16 / 1            |
 
 <div className="image--fixed">
 
-![Large spinner in context example](images/loading-style-4.png)
+![Loading indicator sizes](images/loading-style-size.png)
```

Comment:
This image should show the px/rem in magenta since you are showing it in the table.